signac/contrib/linked_view.py

```diff
@@ -105,7 +149,22 @@ def _update_view(prefix, links, leaf='job'):
 
 
 def _analyze_view(prefix, links, leaf='job'):
-    "Analyze an existing view to prepare for update."
+    """Analyze an existing view to prepare for update.
+
+    Parameters
+    ----------
+    prefix : str
+         (Default value = None).
+    links : dict
+        Linked view.
+    leaf : str
+         (Default value = 'job').
+
+    Returns
+    -------
+    tuple
```

Comment:
Add a description of the variables `obsolete`, `to_update`, and `new`.

Reply:
added, please have a look.